mlflow/utils/file_utils.py

```diff
@@ -13,6 +13,10 @@
 from six.moves import urllib
 
 import yaml
+try:
+    from yaml import CSafeLoader as SafeLoader, CSafeDumper as SafeDumper
+except ImportError:
+    from yaml import SafeLoader, SafeDumper
```

Comment:
Can you use a different alias? Eg.

```
try:
    from yaml import CSafeLoader as YamlSafeLoader, CSafeDumper as YamlSafeDumper
except ImportError:
    from yaml import SafeLoader as YamlSafeLoader, SafeDumper as YamlSafeDumper
```

and use `YamlSafeLoader` and `YamlSafeDumper` in load and dump method.

Reply:
Done. One more question, should I squash the commits?

Reply:
No worries. I can squash commits when merging.